<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
	"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html>
<head>
	<meta http-equiv="Content-type" content="text/html; charset=UTF-8" />
	<title>Steven D. Essinger</title>
	<meta http-equiv="Content-Language" content="en-us" />
	
	<meta http-equiv="imagetoolbar" content="no" />
	<meta name="MSSmartTagsPreventParsing" content="true" />
	
	<meta name="description" content="A Living CV and Portfolio" />
	<meta name="keywords" content="Data Science, Machine Learning, Bioinformatics, Electrical Engineering" />
	
	<meta name="author" content="Steven D. Essinger" />
	
	<style type="text/css" media="all">@import "master.css";</style>  <style type="text/css" media="all">@import "master.css";</style>
</head>

<body>

<div id="page-container">

<div style="position:relative;top:-10px;margin-left:675px">
<img src="drexel.png" style="position:absolute" height="80"
   width="80" alt="Drexel" />
</div>

<img src="signature.png" height="61px" width="400px"/>

<!-- Icon Links -->
<div style="position:relative;top:130px;margin-left:570px">
<div style="position:absolute; width:220px; height:41px">
	<a href="sessinger_resume_050514.pdf" target="_blank">
		<img src="iconResume.png" height="41" width="40" alt="Resume" title="Resume" style="float:left"/></a>
	<a href= "https://www.linkedin.com/in/sessinger" target="_blank">
		<img src="linkedin.png" height="40" width="40" alt="LinkedIn" title="LinkedIn" style="float:left; margin-left:2px"></a>
	<a href="https://github.com/sessinger?tab=repositories" target="_blank">
		<img src="github.png" height="40" width="40" alt="Github" title="Github" style="float:left; margin-left:2px"/></a>
	<a href="http://scholar.google.com/citations?user=eBF3Q14AAAAJ" target="_blank">
		<img src="scholar.png" height="40" width="40" alt="Google Scholar" title="Google Scholar" style="float:left; margin-left:2px"/></a>
	<!--<a href="http://www.kaggle.com/users/136401/alwayssunny" target="_blank">
		<img src="kaggle.png" height="40" width="40" alt="Kaggle" title="Kaggle" style="float:left; margin-left:2px"/></a> -->
</div>
</div>

<div id="header"></div>

<div id="body1">
<p style="font-size:17px"><b>Information</b><p>
<p><a href="mailto:sessinger@drexel.edu" target=*_top*>sessinger@drexel.edu</a><br>
Ecological & Evolutionary Signal Processing & Informatics Laboratory<br>
Electrical & Computer Engineering<br>
Drexel University<br>
3141 Chestnut Street<br>
Philadelphia, PA 19104<p>
</div>

<!-- <div id="body2"> -->

<nav>
	<ul>
		<li><a href="#Publications">Publications</a>
			<ul style="text-align:center">
				<li><a href="#Journal">Journal</a></li>
				<li><a href="#Conference">Conference</a></li>
				<li><a href="#Magazine">Magazine</a></li>
			</ul>
		</li>
		<li><a href="#Projects">Projects</a>
			<ul style="font-size:10px">
				<li><a href="#Thesis"><span style="color:yellow">Ph.D. Thesis:</span> Partitioning Abiotic and Biotic Contributions to Community Variation</a></li>
				<li><a href="#LatentBiotic">Detection of Latent Biotic Covariation Embedded in Environmental Gradient Responses</a></li>
				<li><a href="#ARBToolkit">A Python Toolkit for Custom Databases and Massive Phylogenies</a></li>
				<li><a href="#PSO">Ordering Samples along Environmental Gradients using Particle Swarm Optimization</a></li>
				<li><a href="#NNMetagenomics">Neural Network-based Taxonomic Clustering for Metagenomics</a></li>
				<li><a href="#BenchmarkBLAST">Benchmarking Accuracy of BLAST Taxonomic Classification of Metagenomics Reads</a></li>
				<li><a href="#NovoDisplay">Wireless Mesh-Networked E-Signage System</a></li>
			</ul>
		</li>
		<li><a href="#Patents">Patents</a></li>
		<li><a href="#Teaching">Teaching</a></li>
		<li><a href="#Awards">Awards</a></li>
	</ul>
</nav>

<div id="about">
<a name="About"></a>
<h2><b>About Me</b></h2>
<p style="text-align:justify">I am a recent Ph.D. graduate in the process of transitioning from academia back to industry and am seeking positions in the intersection of <b>data science</b>, <b>machine learning</b> and <b>analytics</b> in the San Francisco Bay Area. For the past five years my attention has been focused on the development of novel methods and end-to-end analytics pipelines for ecologist collaborators. I am enthusiastic about expanding beyond biological applications to broaden my expertise.</p>

<p style="text-align:justify">My work has been highly interdisciplinary with collaboration alongside biologists to assist them in providing insight into their unique datasets. Examples of my deliverables include:</p>

<p><b>Algorithm & Method Development</b>
<li>Coupled regression and non-parametric hypothesis testing for detecting biological interactions <a href="#Thesis"><b>NullSens</b></a></li>
<li>Built an unsupervised machine learning approach for hierarchical classification of DNA sequences <a href="#NNMetagenomics"><b>Look</b></a></li>
<li>Created a particle swarm optimization method for ordering biological samples <a href="#PSO"><b>Look</b></a></li>
</p>

<p><b>End-to-End Data Analysis (Data Munging thru Visualization)</b></p>
<li>Developed Python toolkit to integrate custom databases, alignments and trees for exploratory analyses <a href="#ARBToolkit"><b>Look</b></a></li>
<li>Created analytical pipeline for performing meta-analysis of fish gut communities for ecologist collaborator <a href="Publications/Molecular_Ecology.pdf" target="_blank"><b>PDF</b></a></li>
<li>Critiqued dimensionality reduction tools for biological sample visualization</li>
<li>Routinely performed systematic benchmarking of bioinformatics algorithms; advised best practices <a href="#BenchmarkBLAST"><b>Example</b></a></li>

<p>My workflow has relied predominately on Python and Matlab for their rapid prototyping capabilities.  I currently prefer to use Python in concert with: <b><span style="color:black">iPython notebook, Scipy, Numpy, Scikit-learn, Pandas, Matplotlib, Git</span></b></p>

<p style="text-align:justify">Previously, I assumed a more classical electrical engineering role working as a lead engineer in a multinational R&D group developing a highly-scalable wireless electronic paper retail signage system. The resulting prototype inspired numerous <a href="http://patft.uspto.gov/netacgi/nph-Parser?Sect1=PTO2&Sect2=HITOFF&u=%2Fnetahtml%2FPTO%2Fsearch-adv.htm&r=0&p=1&f=S&l=50&Query=in%2FEssinger+and+an%2FMetrologic&d=PTXT" target="_blank"><b><span style="color:red">patents</span></b></a> and two successful long-term pilot demonstrations with partners both in academia and industry. Other notable projects I have worked on include a refined high frequency modulation circuit to reduce coherent noise in laser illumination applications (patented) and several RFID antennas for handheld barcode scanners. As a prelude to my engineering career, I flew with the FAA in a hollowed-out Convair 440 performing data acquisition on a novel GPS-based aircraft landing system <a href="http://en.wikipedia.org/wiki/Local_Area_Augmentation_System" target="_blank" font-weight="bold">(LAAS)</a>.</p></div>

<div class="line-separator"></div>

<div id="projects-section">
<a name="Projects"></a>
<h2><b><a id="Projects">Projects</a></b></h2>
<a name="Thesis"></a>
<h3 style="text-align:center"><span style="color:red; font-weight:bold">Ph.D. Thesis:</span> Partitioning Abiotic and Biotic Contributions to Community Variation</h3>
<div style="width:483px; height:384px; border-style:solid; border-color:grey; border-width:3px; float:right">
<div style="width:225px; height:384px; float:right; border-color:black; border-style:solid; border-width:0px">
<a href="Publications/thesis_manuscript_final_essinger.pdf" style="float:right; margin-top:5px"><img src="BarThesis.png" alt="Ph.D .Thesis" width="220" height="180" ></a>
<a href="Publications/thesis_manuscript_final_essinger.pdf" style="float:right; margin-top:10px"><img src="PerformanceThesis.png" alt="Ph.D. Thesis" width="220px" height="180px"></a>
</div>
<a href="Publications/thesis_manuscript_final_essinger.pdf" style="float:right; margin-top:25px"><img src="MethodThesis2.png" alt="Ph.D .Thesis" width="255" height="340"></a>
</div>

<div style="border-color:orange; border-style:solid; border-width:3px; width:248px; height:370px; margin-right:490px; padding:7px; text-align:justify; font-family:Ariel; font-size:15px">
&#8226; &#8226; <a href="Publications/thesis_manuscript_final_essinger.pdf"><b><span style="color:blue">Thesis</span></b></a>	
&#8226; &#8226; <a href="https://github.com/sessinger/NullSens" target="_blank"><b><span style="color:blue">NullSens (R-Package)</span></b></a>
&#8226; &#8226; The interaction among organisms and their environment is a predominant topic of interest to ecologists. To study community responses to environmental factors, ordination and regression techniques are typically employed; however, for studying species interactions, methods primarily rely on analyzing patterns of presence/absence. Each of these types of analyses are carried out independently because there is a lack of unified statistical methods for simultaneous analysis of biotic and abiotic factors influencing community composition. This thesis presents a unified method that first removes environmentally explained variation from species responses so that apparent species interactions are not masked or augmented by the abiotic responses, thus partitioning the factors.
</div>
<!-- <div style="background:grey; clear:both; width:760px; height:150px; margin-top:2px"></div> -->
<div></div>

<a name="LatentBiotic"></a>
<h3 style="text-align:center; margin-top:50px">Detection of Latent Biotic Covariation Embedded in Environmental Gradient Responses</h3>
<p><a href="Posters/Biotic_Covariation.pdf" style="float:left">
<img src="Biotic_Covariation.jpg" alt="Null Model Poster" width="486" height="375"></a><p>
<div style="border-color:orange; border-style:solid; border-width:3px; width:248px; height:355px; margin-left:492px; padding:7px; text-align:justify; font-family:Ariel; font-size:15px">The interaction among organisms and their environment is a predominant topic of interest to ecologists. Both environmental factors and species interactions among organisms structure communities. The relative importance of these factors in influencing community composition is openly debated and may have profound implications for management of communities and our ability to predict community responses to perturbations. There is currently no unified statistical method for simultaneous analysis of environmental and biotic factors influencing community composition. This work seeks to model ecological communities and partition the variation explained by the environmental and biotic factors respectively. This work is a prelude to my <a href="Publications/thesis_manuscript_final_essinger.pdf" font-weight="bold"><b>thesis</b></a>.
</div>
<!-- <div style="background:grey; clear:both; width:760px; height:150px; margin-top:2px"></div> -->

<a name="ARBToolkit"></a>
<h3 style="text-align:center; margin-top:50px">A Python Toolkit for Custom Databases and Massive Phylogenies</h3>
<p><a href="Posters/ARBPython.pdf" style="float:right; border-color:grey; border-width:2px; border-style:solid">
<img src="ARBPython.png" alt="ARB Python Poster" width="484" height="375"></a></p>
<div style="border-color:orange; border-style:solid; border-width:3px; width:248px; height:359px; margin-right:490px; padding:7px; text-align:justify; font-family:Ariel; font-size:15px">Computational approaches employed by ecologists for analyzing sequence data (e.g. alignment, phylogeny) typically scale nonlinearly in execution time with the size of the dataset. This often serves as a bottleneck for processing experimental data. To keep up with experimental data demands, ecologists are forced to choose between continually upgrading costly in-house computer hardware or outsourcing the most demanding computations to the cloud. To mitigate this tradeoff we introduce a script-based pipeline to leverage the utility of the interactive exploratory tools offered by the desktop tool ARB with the computational throughput of cloud-based resources. Our pipeline serves as middleware between the desktop and the cloud. Code can be found <a href="https://github.com/groovyjawn/ARB-toolkit" font-weight="bold"><b>here</b></a>.
</div>
<!-- <div style="background:grey; clear:both; width:760px; height:150px; margin-top:2px"></div> -->

<a name="PSO"></a>
<h3 style="text-align:center; margin-top:50px">Ordering Samples along Environmental Gradients using Particle Swarm Optimization</h3>
<p><a href="Posters/PSO.pdf" style="float:left">
<img src="PSO.png" alt="PSO Poster" width="486" height="375"></a></p>
<div style="border-color:orange; border-style:solid; border-width:3px; width:248px; height:355px; margin-left:492px; padding:7px; text-align:justify; font-family:Ariel; font-size:15px">Ecologists are often concerned with how the environment structures communities of organisms. Changes in environmental conditions over space or time are known as gradients. Community variation along gradients are therefore of predominant interest. Methods for inferring gradients seek to order sample sites along the direction of the most dominant gradient. Techniques such as PCA are employed, but suffer from artifacts that skew the results. This study takes a novel approach to ordering samples by treating it like a "traveling salesman problem". PSO is employed to move through the solution space. The Wald-Wolfowitz test is used to determine if the sites are ordered correctly based on the species' abundance curves. More detail can be found <a href="Publications/Ordering_Samples_PSO.pdf" font-weight="bold"><b>here</b></a>.
</div>
<!-- <div style="background:grey; clear:both; width:760px; height:150px; margin-top:2px"></div> -->

<a name="NNMetagenomics"></a>
<h3 style="text-align:center; margin-top:50px">Neural Network-based Taxonomic Clustering for Metagenomics</h3>
<div style="width:480px; height:377px; border-style:solid; border-width:0px; float:right">
	<a href="Posters/NN_ISME.pdf" style="float:left; border-color:grey; border-style:solid; border-width:1px; margin-left:20px">
		<img src="NN_ISME.png" alt="NN ISME Poster" width="266px" height="375px"></a>
	<a href="Posters/NN_Pseudocode.pdf" style="margin-left:20px">
		<img src="NN_Pseudocode.png" alt="NN ISME Poster" width="150px" height="375px"></a>
</div>
<div style="border-color:orange; border-style:solid; border-width:3px; width:251px; height:357px; margin-left:15px; padding:7px; text-align:justify; font-family:Ariel; font-size:15px">Due to the proliferation of sequencing technology, biologist are generating massive amounts of DNA sequences. The likelihood of a query sequence originating from a reference genome may be provided by the posterior probability of the Naive Bayes Classifier (NBC). Since most query sequences are novel, this study investigates the use of NBC posteriors as features for unsupervised clustering of novel sequences. K-means and Adaptive Resonance Theory (ART) are considered for classifiers. The main advantage of ART is that the number of clusters does not need to be specified a priori as in K-means. Rather a vigilance parameter is set to control the formation of a new cluster and is dependent on the data being clustered. More detail can be found <a href="Publications/Neural_Network_Classification.pdf" font-weight="bold"><b>here</b></a>. 
</div>
<!-- <div style="background:grey; clear:both; width:760px; height:150px; margin-top:2px"></div> -->

<a name="BenchmarkBLAST"></a>
<h3 style="text-align:center; margin-top:50px">Benchmarking Accuracy of BLAST Taxonomic Classification of Metagenomics Reads</h3>
<div style="width:502px; height:377px; border-style:solid; border-width:0px; float:left">
	<a href="Posters/BLAST_RECOMB.pdf" style="float:left; border-style:solid; border-color:grey; border-width:1px; margin-left:15px">
		<img src="BLAST_RECOMB.png" alt="BLAST Benchmark Poster" width="266" height="375"></a>
	<a href="BLAST_RECOMB_Fig1.png" style="float:left; margin-left:20px; margin-top:5px">
		<img src="BLAST_RECOMB_Fig1.png" alt="Experimental Setup 1" width="180" height="200"></a>
	<a href="BLAST_RECOMB_Fig2.png" style="float:left; margin-left:20px; margin-top:10px">
		<img src="BLAST_RECOMB_Fig2.png" alt="Experimental Setup 2" width="180" height="160"></a>
	
</div>
<div style="border-color:orange; border-style:solid; border-width:3px; width:248px; height:355px; margin-left:492px; padding:7px; text-align:justify; font-family:Ariel; font-size:15px">Classification of novel DNA sequences to bacterial <a href="http://en.wikipedia.org/wiki/Biological_classification" font-weight="bold" target="_blank">taxonomy</a> is an integral task for researchers in numerous areas of biology. <a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" font-weight="bold" target="_blank"><b>BLAST</b><a>, a supervised learning method, has widespread use in searching for closest matches based on sequence similarity to known organisms. More than 95% of bacteria are not in the databases, however. While effective on known organisms, it is unknown how accurate BLAST classification is on novel sequences. This study investigates BLAST accuracy on datasets with under-represented genomes in the database and errors in the query sequences. More detail can be found <a href="Publications/Benchmarking_BLAST.pdf" font-weight="bold" target="_blank"><b>here</b></a> and <a href="Publications/Sequence_Error_BLAST.pdf" font-weight="bold" target="_blank"><b>here</b></a>. The study has also been expanded to investigate other methods such as the Naive Bayes Classifier, <a href="Publications/Comparison_Statistical_Metagenomics.pdf" font-weight="bold" target="_blank"><b>here</b></a>.
</div>
<!-- <div style="background:grey; clear:both; width:760px; height:150px; margin-top:2px"></div> -->

<a name="NovoDisplay"></a>
<h3 style="text-align:center; margin-top:50px">Wireless Mesh-Networked E-Signage System</h3>
<div style="width:483px; height:384px; border-style:solid; border-color:grey; border-width:3px; float:right">
<div style="width:483px; height:170px; float:right; border-color:black; border-style:solid; border-width:0px">
<a href="novo_system.png" style="float:right; margin-top:0px"><img src="novo_system.png" alt="NOVOdisplay" width="483" height="170" ></a>
</div>
<a href="Esign.png" style="float:right; margin-top:0px"><img src="Esign.png" alt="E-Sign" width="483px" height="214px"></a>
</div>
<div style="border-color:orange; border-style:solid; border-width:3px; width:248px; height:370px; margin-right:490px; padding:7px; text-align:justify; font-family:Ariel; font-size:15px"> While working in the R&D department at Metrologic (now Honeywell) I worked up to the Team Lead position, developing an electronic paper retail signage system supported by a Zigbee mesh-network communication system. I authored a brief white paper describing the product, which may be found <a href="Publications/NOVOdisplay_ESign_Platform_Whitepaper.pdf" font-weight="bold" target="_blank"><b>here</b></a>. Working in our 10 person globally-distributed team we developed several working prototypes for potential customers and generated numerous patents based on our work. Relevant <b>patents</b> may be found <a href="http://appft.uspto.gov/netacgi/nph-Parser?Sect1=PTO2&Sect2=HITOFF&u=%2Fnetahtml%2FPTO%2Fsearch-adv.html&r=0&p=1&f=S&l=50&Query=in%2Fessinger+and+an%2Fmetrologic+and+abst%2Fwireless&d=PG01" font-weight="bold" target="_blank"><b>here</b></a> and <a href="http://patft.uspto.gov/netacgi/nph-Parser?Sect1=PTO2&Sect2=HITOFF&u=%2Fnetahtml%2FPTO%2Fsearch-adv.htm&r=0&p=1&f=S&l=50&Query=in%2Fessinger+and+an%2Fmetrologic+and+abst%2Fwireless&d=PTXT" font-weight="bold" target="_blank"><b>here</b></a>. My direct responsibilities included the systems design and drafting of the figures describing the system, developing IP, the RF hardware design, coordination of all technical development and supporting marketing and sales with customer demonstrations.  
</div>
</div>
<p></p>

<div class="line-separator"></div>
	
<div id="publications">
<a name="Publications"></a>
<h2><b><a id="Pub">Publications</a></b></h2>
<a name="Journal"></a>
<h3 style="color:red">Journal</h3>
<p>Essinger, S., Blackwood, C., Rosen, G., "<b>NullSens: Partitioning Abiotic and Biotic Contributions to Community Variation,</b>" Ecology, 2014. (Submission)</p>
<p>Essinger, S., Reichenberger, E., Blackwood, C., Rosen, G., "<b>A Python Toolkit for ARB to Integrate Custom Databases and Externally-built Phylogenies,</b>" PLOS ONE, 2014. (Submission)</p>
<p>Sullam, K., Essinger, S., Lozupone, C., O'Connor, M., Rosen, G., Knight, R., Kilham, S., Russell, J., "<b>Environmental and ecological factors that shape the gut bacterial communities of fish: a meta-analysis,</b>" Molecular Ecology, 2012. <a href="Publications/Molecular_Ecology.pdf">PDF</a> </p>
<p>Rosen, G., Polikar, R., Diamantino, C., Essinger, S., and Sokhansanj, B., "<b>Discovering the Unknown: Improving Detection of Novel Species and Genera from Short Reads,</b>" Journal of Biomedicine and Biotechnology, Jan. 2011. <a href="Publications/Discovering_the_Unknown.pdf">PDF</a> </p>
<p>Rosen, G. and Essinger, S., "<b>Comparison of Statistical Methods to Classify Environmental Genomic Fragments,</b>" IEEE Transactions on Nanobioscience, Sep. 2010, pp. 1-7. <a href="Publications/Comparison_Statistical_Metagenomics.pdf">PDF</a> </p>
<p>Rosen, G., Sokhansanj, B., Polikar, R., Bruns, M.A., Russell, J., Garbarine, E., Essinger, S., and Yok N., "<b>Signal Processing for Metagenomics: Extracting Information from the Soup,</b>" Current Genomics, Nov. 2009. <a href="Publications/Signal_Processing_for_Metagenomics.pdf">PDF</a> </p>

<a name="Conference"></a>
<h3 style="color:red">Conference: Peer Reviewed with Oral Presentation</h3>
<p>Essinger, S., and Rosen, G., "<b>Ordering Samples Along Environmental Gradients using Particle Swarm Optimization,</b>" IEEE EMBC Conference, Boston MA, August 2011. <a href="Publications/Ordering_Samples_PSO.pdf">PDF</a> </p>
<p>Essinger, S. and Rosen, G., "<b>An Introduction to Machine Learning for Students in Secondary Education,</b>" IEEE Signal Processing in Education Workshop, January 2011. <b><span style="color:red">(Best Student Paper Award)</span></b>  <a href="Publications/Introduction_to_Machine_Learning.pdf">PDF</a> </p>
<p>Essinger, S., Polikar, R. and Rosen, G., "<b>Neural Network-based Taxonomic Classification for Metagenomics,</b>" IEEE International Joint Conference on Neural Networks, July 2010. <b><span style="color:red">(Student Travel Award)</span></b>  <a href="Publications/Neural_Network_Classification.pdf">PDF</a> </p>
<p>Essinger, S. and Rosen, G., "<b>The Effect of Sequence Error and Partial Training Data on BLAST Accuracy of Short Reads,</b>" IEEE Bioinformatics and Bioengineering Conference (BIBE), June 2010. <a href="Publications/Sequence_Error_BLAST.pdf">PDF</a> </p>
<p>Essinger, S., Coote, R., Konstantopolous, P., Silverman, J. and Rosen, G., "<b>Reflections and Measures of STEM Teaching and Learning on K-12 Creative And Performing Arts Students,</b>" ASEE Annual Conference, June 2010.  <a href="Publications/Reflections_Measures_STEM.pdf">PDF</a> </p>
<p>Essinger, S. and Rosen, G., "<b>Benchmarking BLAST Accuracy of Genus/Phyla Classification of Metagenomic Reads,</b>" Pacific Symposium on Biocomputing, Jan. 2010. <b><span style="color:red">(Student Travel Award)</span></b> <a href="Publications/Benchmarking_BLAST.pdf">PDF</a> </p>

<a name="Magazine"></a>
<h3 style="color:red">Book Chapters & Magazines</h3>
<p>Rosen, G.L., Silverman, J., and Essinger, S.D., "<b>Inquiry-Based Learning Through Image Processing,</b>" IEEE Signal Processing Magazine, January 2012. <a href="Publications/Inquiry_Based_Learning.pdf">PDF</a> </p>
<p>Bouchot, J.L., Trimble, W., Ditzler, G., Lan, Y., Essinger, S.D. and Rosen, G.L., "<b>Advances in Machine Learning for Processing and Comparison of Metagenomic Data,</b>" Computational Systems Biology, Ed. Andres Kriete, Ed. Roland Elis: Academic Press, 2013. (Preprint) <a href="Publications/metagenomics_advances_ML_preprint.pdf">PDF</a> </p>
</div>	
	
<div class="line-separator"></div>

<div id="patents">
<a name="Patents"></a>
<h2><b><a id="Patents">Patents</a></b></h2>
<p style="color:red"><b>U.S. PATENTS SELECTED</b></p>
<p><b><a href="http://appft1.uspto.gov/netacgi/nph-Parser?Sect1=PTO2&Sect2=HITOFF&u=%2Fnetahtml%2FPTO%2Fsearch-adv.html&r=0&p=1&f=S&l=50&Query=in%2FEssinger+and+an%2FMetrologic&d=PG01">PENDING (23)</a>, <a href="http://patft.uspto.gov/netacgi/nph-Parser?Sect1=PTO2&Sect2=HITOFF&u=%2Fnetahtml%2FPTO%2Fsearch-adv.htm&r=0&p=1&f=S&l=50&Query=in%2FEssinger+and+an%2FMetrologic&d=PTXT">GRANTED (22)</b></a> → USPTO Patent Search (in/Essinger and an/Metrologic)</p>

<p><b>Essinger, et al.<span style="padding-left:15em">8,457,013</span><span style="padding-left:15em">June 4, 2013</span></b><br>
Methods of and apparatus for programming and managing diverse network components, including electronic-ink based display devices, in a mesh-type wireless communication network</p>

<p><b>Knowles, et al.<span style="padding-left:15em">7,793,841</span><span style="padding-left:12em">September 14, 2010</span></b><br>
Laser illumination beam generation system employing despeckling of the laser beam using high-frequency modulation of the laser diode current and optical multiplexing of the component laser beams</p>
       
<p><b>Essinger, et al.<span style="padding-left:13em">US2010/0177707 A1</span><span style="padding-left:12em">July 15, 2010</span></b><br>
Method of and apparatus for increasing the SNR at the RF antennas of Wireless end-devices on a wireless communication network, while minimizing the RF power transmitted by the wireless coordinator and routers</p>
</div>

<div class="line-separator"></div>

<div id="teaching">
<a name="Teaching"></a>
<h2><b><a id="Teaching">Lectures & Teaching</a></b></h2>
<table style="font-family:Jester; color: #000; font-size:16px; width:100%"> 
<tr>
<td style="width: 75%">TA: Transform Methods</td>
<td>September-December 2013</td>
</tr>
<tr>
<td>TA: Electrical Engineering Labs</td>
<td>March-September 2013</td>
</tr>
<tr>
<td>TA: Dynamic Engineering Systems</td>
<td>January-March 2013</td>
</tr>
<tr>
<td>TA: Linear Engineering Systems</td>
<td>September-December 2012</td>
</tr>
<tr>
<td>Maximum Likelihood for Phylogeny (Guest lecturer, Drexel Bioinformatics)</td>
<td>February 2012</td>
</tr>
<tr>
<td>Phylogenetic Methods (Guest lecturer, Rowan University)</td>
<td>November 2011</td>
</tr>
<tr>
<td>NSF Discovery K-12 Fellow: Develop and Teach STEM labs for H.S. Student</td>
<td>September 2008-10</td>
</tr>
<tr>
<td  style="padding-left:2em">Creative and Performing Arts (CAPA) High School, Philadelphia, PA</td>
<td>2008-09</td>
</tr>
<tr>
<td style="padding-left:2em">Philadelphia High School for Girls, Philadelphia, PA</td>
<td>2009-10</td>
</tr>
<tr>
<td>Phylogenetic Trees (Guest lecturer, Drexel undergraduate bioinformatics)</td>
<td>February 2009</td>
</tr>
<tr>
<td>Bioinformatics and Signal Processing (CAPA H.S., lecture and lab series)</td>
<td>January 2009</td>
</tr>
<tr>
<td>Johns Hopkins Center for Talented Youth (Bioinformatics Lecturer)</td>
<td>October 2008</td>
</tr>
</table>
</div>	

<div class="line-separator"></div>


<div id="awards">
<a name="Awards"></a>
<h2><b><a id="Awards">Awards</a></b></h2>
<table style="font-family:Jester; color: #000; font-size:16px; width: 100%"> 
<tr>
<td style="width: 75%">Best Research Poster (3rd): IEEE Research Symposium, Drexel University, $100</td>
<td>March 2013 </td>
</tr>
<tr>
<td>Best Student Paper Award: IEEE Signal Processing Conference, Sedona, AZ, $500</td>
<td>January 2011</td>
</tr>
<tr>
<td>NSF Discover K-12 Research Fellowship, $24,000 per year + tuition</td>
<td>September 2009-10</td>
</tr>
<tr>
<td>Travel Grant to IEEE World Congress on Computational Intelligence, $800</td>
<td>July 2010</td>
</tr>
<tr>
<td>Best Research Poster Award (1st): IEEE Graduate Forum, Drexel University, $500</td>
<td>March 2010</td>
</tr>
<tr>
<td>Travel Grant to Pacific Symposium on Biocomputing (PSB), $1,200</td>
<td>January 2010</td>
</tr>
<tr>
<td>NSF Fellow Biocomplexity Summer School, Istanbul Turkey, $2,200</td>
<td>July 2009</td>
</tr>
<tr>
<td>Eta Kappa Nu Electrical Engineering Honor Society</td>
<td>September 2006</td>
</tr>
<tr>
<td>Kappa Theta Epsilon Honor Society</td>
<td>September 2005</td>
</tr>
</table>
</div>

<div id="footer">
	<div id="altnav">
		<a href="#About">About</a> - 
		<a href="#Publications">Publications</a> - 
		<a href="#Projects">Projects</a> - 
		<a href="#Patents">Patents</a> - 
		<a href="#Teaching">Teaching</a> -
		<a href="#Awards">Awards</a>

	</div>
	Copyright © 2014 Steven D. Essinger

</div>
</div>

</body>
</html>
