<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta name="description" content="A living resume and CV to help me feel better about myself for when I'm in the thick of things.">
    <meta name="keywords" content="Data Science, Machine Learning, AI, Artificial Intelligence, Engineering, Leadership">
    <meta name="author" content="Steve Essinger">
    <title>Steve Essinger | Applied Scientist & Engineer </title>
    <link rel="stylesheet" href="index.css">
</head>

<body>
<header id="top-header">
    <div class="header-content">
        <nav>
            <ul>
                <li><a href="#publications">Publications</a></li>
                <li><a href="#projects-section">Thesis</a></li>
            </ul>
        </nav>
       <img src="images/profile-pic.jpg" alt="Steve Essinger" class="profile-pic">
       <h1>Steve Essinger</h1>
        <p>
            <b>Connect via </b><a href="https://linkedin.com/in/sessinger"><b>LinkedIn</b></a><br>
        </p>
        </div>
</header>

<div id="page-container">

<section id="quickthought">
    <h2>App Development</h2>
        <p><b>I built an app end-to-end from scratch - Now in alpha release!</b></p>
        <div class="quickthought">
            <a href="http://quickthought.io"><img src="images/quickthought.png" alt="QuickThought logo" class="quickthought-img"></a>
            <p><i>Your thoughts, easily captured, effortlessly organized.</i></p>
        </div>
        <p>QuickThought lets you easily capture your thoughts via speech-to-text or typing. It then automatically classifies your thought by type (e.g. reminder, summary, idea, etc.) and topic (e.g. family, work, house, finances etc.). All your thoughts are neatly filed away for when you're ready to recall them. Super simple, highly effective!</p>
</section>

<section id="publications">
    <h2>Publications, Patents & Talks</h2>
    <p><b>Additional @ </b>
        <b><a href="http://scholar.google.com/citations?user=eBF3Q14AAAAJ">Google Scholar</a></b>
    </p><br>
    <article>
        <p>Essinger, S. <b>Product Recommendations at Nike.</b> Inaugural Nike Sport+AI Conference, Portland, Oregon, 2022. <a href="https://sportai.splashthat.com">splash</a></p>
    </article>
    <article>
        <p>Essinger, S., Huber, D., and Tang, D., <b>AIR: Personalized Product Recommender System for Nike’s Digital Transformation.</b> In Fifteenth ACM Conference on Recommender Systems (RecSys ’21), September 27–October 01, 2021, Amsterdam, Netherlands. ACM, New York, NY, USA <a href="https://dl.acm.org/doi/10.1145/3460231.3474621">pub, </a><a href="https://www.youtube.com/watch?v=3QbMbnTQres">rec</a></p>
    </article>
    <article>
        <p>Schneck, Essinger, et al., <b>Cold-Start Podcast Recommendations: A system and method are disclosed for recommending podcasts.</b> US-20200007917-A1, January 2, 2020</p>
    </article>
    <article>
        <p>Abdollahpouri, H., Essinger, S., <b>“Towards Effective Exploration/Exploitation in Sequential Music Recommendation,"</b> ACM Conference on Recommender Systems, Como, Italy, 2017. <a href="https://arxiv.org/abs/1812.03226">arxiv</a> </p>
    </article>
    <article>
        <p>Abdollahpouri, H., Essinger, S., <b>"Multiple Stakeholders in Music Recommender Systems,"</b> ACM Conference on Recommender Systems, VAMS Workshop, Como, Italy, 2017. <a href="https://arxiv.org/abs/1708.00120">arxiv</a> </p>
    </article>
    <article>
        <p>Essinger, S. <b>Life at Pandora: Spotlight on Computational Programming,</b> Pandora Media, Inc., 2017 <a href="https://www.youtube.com/watch?v=izN1dpYrCLo">YouTube</a></p>
    </article>
    <article>
        <p>Essinger, S. <b>Data Driven Decision Making,</b> Stanford Graduate School of Business, Lecturer, (MGTECON536), 2016 & 2017</p>
    </article>
    <article>
        <p>Essinger, S., Reichenberger, E., Blackwood, C., Rosen, G., "<b>A Python Toolkit for ARB to Integrate Custom Databases and Externally-built Phylogenies,</b>" PLOS ONE, 2014. <a href="Publications/toolkit-for-arb-plosone.pdf">pdf</a></p>
    </article>
    <article>
        <p>Essinger, S. <b>Partitioning Abiotic and Biotic Contributions to Community Variation</b> PhD Thesis, Drexel University, 2013. <a href="Publications/thesis_manuscript_final_essinger.pdf">pdf</a></p>
    </article>
    <article>
        <p>Bouchot, J.L., Trimble, W., Ditzler, G., Lan, Y., Essinger, S.D. and Rosen, G.L., "<b>Advances in Machine Learning for Processing and Comparison of Metagenomic Data,</b>" Computational Systems Biology, Ed. Andres Kriete, Ed. Roland Elis: Academic Press, 2013. (Preprint) <a href="Publications/metagenomics_advances_ML_preprint.pdf">pdf</a> </p>
    </article>
    <article>
        <p>Essinger, et al. <b>Methods of and apparatus for programming and managing diverse network components, including electronic-ink based display devices, in a mesh-type wireless communication network</b> USPTO 8,457,013, June 2013</p>
    </article>
    <article>
        <p>Sullam, K., Essinger, S., Lozupone, C., O'Connor, M., Rosen, G., Knight, R., Kilham, S., Russell, J., "<b>Environmental and ecological factors that shape the gut bacterial communities of fish: a meta-analysis,</b>" Molecular Ecology, 2012. <a href="Publications/Molecular_Ecology.pdf">pdf</a> </p>
    </article>
    <article>
        <p>Rosen, G.L., Silverman, J., and Essinger, S.D., "<b>Inquiry-Based Learning Through Image Processing,</b>" IEEE Signal Processing Magazine, January 2012. <a href="Publications/Inquiry_Based_Learning.pdf">pdf</a> </p>
    </article>
    <article>
        <p>Essinger, S., and Rosen, G., "<b>Ordering Samples Along Environmental Gradients using Particle Swarm Optimization,</b>" IEEE EMBC Conference, Boston MA, August 2011. <a href="Publications/Ordering_Samples_PSO.pdf">pdf</a> </p>
    </article>
    <article>
        <p>Essinger, S. and Rosen, G., "<b>An Introduction to Machine Learning for Students in Secondary Education,</b>" IEEE Signal Processing in Education Workshop, January 2011. <a href="Publications/Introduction_to_Machine_Learning.pdf">pdf</a> </p>
    </article>
    <article>
        <p>Rosen, G., Polikar, R., Diamantino, C., Essinger, S., and Sokhansanj, B., "<b>Discovering the Unknown: Improving Detection of Novel Species and Genera from Short Reads,</b>" Journal of Biomedicine and Biotechnology, Jan. 2011. <a href="Publications/Discovering_the_Unknown.pdf">pdf</a> </p>
    </article>
    <article>
        <p>Essinger, et al. <b>Method of and apparatus for increasing the SNR at the RF antennas of Wireless end-devices on a wireless communication network, while minimizing the RF power transmitted by the wireless coordinator and routers</b> USPTO US2010/0177707 A1, July 2010</p>
    </article>
    <article>
        <p>Rosen, G. and Essinger, S., "<b>Comparison of Statistical Methods to Classify Environmental Genomic Fragments,</b>" IEEE Transactions on Nanobioscience, Sep. 2010, pp. 1-7. <a href="Publications/Comparison_Statistical_Metagenomics.pdf">pdf</a> </p>
    </article>
    <article>
        <p>Essinger, S., Polikar, R. and Rosen, G., "<b>Neural Network-based Taxonomic Classification for Metagenomics,</b>" IEEE International Joint Conference on Neural Networks, July 2010. <b><span style="color:red"></span></b>  <a href="Publications/Neural_Network_Classification.pdf">pdf</a> </p>
    </article>
    <article>
        <p>Essinger, S. and Rosen, G., "<b>The Effect of Sequence Error and Partial Training Data on BLAST Accuracy of Short Reads,</b>" IEEE Bioinformatics and Bioengineering Conference (BIBE), June 2010. <a href="Publications/Sequence_Error_BLAST.pdf">pdf</a> </p>
    </article>
    <article>
        <p>Essinger, S., Coote, R., Konstantopolous, P., Silverman, J. and Rosen, G., "<b>Reflections and Measures of STEM Teaching and Learning on K-12 Creative And Performing Arts Students,</b>" ASEE Annual Conference, June 2010.  <a href="Publications/Reflections_Measures_STEM.pdf">pdf</a> </p>
    </article>
    <article>
        <p>Essinger, S. and Rosen, G., "<b>Benchmarking BLAST Accuracy of Genus/Phyla Classification of Metagenomic Reads,</b>" Pacific Symposium on Biocomputing, Jan. 2010. <b><span style="color:red"></span></b> <a href="Publications/Benchmarking_BLAST.pdf">pdf</a> </p>
    </article>
    <article>
        <p>Knowles, et al. <b>Laser illumination beam generation system employing despeckling of the laser beam using high-frequency modulation of the laser diode current and optical multiplexing of the component laser beams,</b>  USPTO# 7,793,841. Sept. 2010</p>
    </article>
    <article>
        <p>Rosen, G., Sokhansanj, B., Polikar, R., Bruns, M.A., Russell, J., Garbarine, E., Essinger, S., and Yok N., "<b>Signal Processing for Metagenomics: Extracting Information from the Soup,</b>" Current Genomics, Nov. 2009. <a href="Publications/Signal_Processing_for_Metagenomics.pdf">pdf</a> </p>
    </article>
</section>

    <div class="line-separator"></div>

    <section id="projects-section">
        <h2>Thesis & Graduate Work</h2>
<!--        <div class="graduate-header">-->
<!--            <img src="images/myphoto.jpg" alt="Grad School Profile Picture" />-->
<!--            <div class="graduate-header-info">-->
<!--                <b>Ecological & Evolutionary Signal Processing</br> & Informatics Laboratory</b><br>-->
<!--                Electrical & Computer Engineering<br>-->
<!--                Drexel University<br>-->
<!--                3141 Chestnut Street<br>-->
<!--                Philadelphia, PA 19104<p>-->
<!--            </div>-->
<!--        </div>-->

        <div class="graduate-intro">
            <p>In graduate school at Drexel University my attention has been focused on the development of novel methods and end-to-end analytics pipelines for molecular ecologists. My work has been highly interdisciplinary with collaboration alongside biologists to assist them in providing insight into their unique genomic datasets. The follow projects are examples my work. </p>
<!--            <p><b>Algorithm & Method Development</b>-->
<!--                <li>Coupled regression and non-parametric hypothesis testing for detecting biological interactions <a href="#Thesis"><b>NullSens</b></a></li>-->
<!--                <li>Built an unsupervised machine learning approach for hierarchical classification of DNA sequences <a href="#NNMetagenomics"><b>Look</b></a></li>-->
<!--                <li>Created a particle swarm optimization method for ordering biological samples <a href="#PSO"><b>Look</b></a></li>-->
<!--            </p>-->

<!--            <p><b>End-to-End Data Analysis (Data Munging thru Visualization)</b></p>-->
<!--                <li>Developed Python toolkit to integrate custom databases, alignments and trees for exploratory analyses <a href="#ARBToolkit"><b>Look</b></a></li>-->
<!--                <li>Created analytical pipeline for performing meta-analysis of fish gut communities for ecologist collaborator <a href="Publications/Molecular_Ecology.pdf" target="_blank"><b>PDF</b></a></li>-->
<!--                <li>Critiqued dimensionality reduction tools for biological sample visualization</li>-->
<!--                <li>Routinely performed systematic benchmarking of bioinformatics algorithms; advised best practices <a href="#BenchmarkBLAST"><b>Example</b></a></li>-->
        </div>

        <article id="thesis-project">
            <h3>Ph.D. Thesis: Partitioning Abiotic and Biotic Contributions to Community Variation</h3>
                <figure class="project-figure">
                    <a href="Publications/thesis_manuscript_final_essinger.pdf">
                        <img src="images/thesis-image.png" alt="Ph.D Thesis Image">
                    </a>
                </figure>
                <div class="project-description">
                    <p>
                        The interaction among organisms and their environment is a predominant topic of interest to ecologists. To study community responses to environmental factors, ordination and regression techniques are typically employed; however, for studying species interactions, methods primarily rely on analyzing patterns of presence/absence. Each of these types of analyses are carried out independently because there is a lack of unified statistical methods for simultaneous analysis of biotic and abiotic factors influencing community composition. This thesis presents a unified method that first removes environmentally explained variation from species responses so that apparent species interactions are not masked or augmented by the abiotic responses, thus partitioning the factors.
                        <a href="Publications/thesis_manuscript_final_essinger.pdf"><b>Read Thesis</b></a>
                    </p>
                </div>
        </article>

        <article id="latent-biotic-project">
            <h3>Detection of Latent Biotic Covariation Embedded in Environmental Gradient Responses</h3>
            <div class="project-content">
                <figure class="project-figure">
                    <a href="Posters/Biotic_Covariation.pdf">
                        <img src="images/biotic-covariation.png" alt="Latent Biotic Covariation Poster">
                    </a>
                </figure>
                <div class="project-description">
                    <p>
                        The interaction among organisms and their environment is a predominant topic of interest to ecologists. Both environmental factors and species interactions among organisms structure communities. The relative importance of these factors in influencing community composition is openly debated and may have profound implications for management of communities and our ability to predict community responses to perturbations. There is currently no unified statistical method for simultaneous analysis of environmental and biotic factors influencing community composition. This work seeks to model ecological communities and partition the variation explained by the environmental and biotic factors respectively. See
                        <a href="Publications/thesis_manuscript_final_essinger.pdf"><b>thesis</b></a>.
                    </p>
                </div>
            </div>
        </article>

        <article id="arb-python-project">
            <h3>A Python Toolkit for Custom Databases and Massive Phylogenies</h3>
            <div class="project-content">
                <figure class="project-figure">
                    <a href="Posters/ARBPython.pdf">
                        <img src="images/arb-python.png" alt="ARB Python Poster">
                    </a>
                </figure>
                <div class="project-description">
                    <p>
                        Computational approaches employed by ecologists for analyzing sequence data (e.g. alignment, phylogeny) typically scale nonlinearly in execution time with the size of the dataset. This often serves as a bottleneck for processing experimental data. To keep up with experimental data demands, ecologists are forced to choose between continually upgrading costly in-house computer hardware or outsourcing the most demanding computations to the cloud. To mitigate this tradeoff we introduce a script-based pipeline to leverage the utility of the interactive exploratory tools offered by the desktop tool ARB with the computational throughput of cloud-based resources. Our pipeline serves as middleware between the desktop and the cloud. Code can be found
                        <a href="https://github.com/sessinger/ARB-toolkit" font-weight="bold"><b>here</b></a>.
                    </p>
                </div>
            </div>
        </article>

        <article id="environmental-gradients">
            <h3>Ordering Samples along Environmental Gradients using Particle Swarm Optimization</h3>
            <div class="project-content">
                <figure class="project-figure">
                    <a href="Posters/PSO.pdf">
                        <img src="images/environmental-gradients.png" alt="Environmental Gradients Poster">
                    </a>
                </figure>
                <div class="project-description">
                    <p>
                        Ecologists are often concerned with how the environment structures communities of organisms. Changes in environmental conditions over space or time are known as gradients. Community variation along gradients are therefore of predominant interest. Methods for inferring gradients seek to order sample sites along the direction of the most dominant gradient. Techniques such as PCA are employed, but suffer from artifacts that skew the results. This study takes a novel approach to ordering samples by treating it like a "traveling salesman problem". PSO is employed to move through the solution space. The Wald-Wolfowitz test is used to determine if the sites are ordered correctly based on the species' abundance curves. More detail can be found
                        <a href="Publications/Ordering_Samples_PSO.pdf" font-weight="bold"><b>here</b></a>.
                    </p>
                </div>
            </div>
        </article>

        <article id="nn-metagenomics">
            <h3>Neural Network-based Taxonomic Clustering for Metagenomics</h3>
            <div class="project-content">
                <figure class="project-figure">
	                <a href="Posters/NN_ISME.pdf">
		                <img src="images/nn-metagenomics.png" alt="NN ISME Poster">
                    </a>
                </figure>
                <div class="project-description">
                    <p>
                        Due to the proliferation of sequencing technology, biologist are generating massive amounts of DNA sequences. The likelihood of a query sequence originating from a reference genome may be provided by the posterior probability of the Naive Bayes Classifier (NBC). Since most query sequences are novel, this study investigates the use of NBC posteriors as features for unsupervised clustering of novel sequences. K-means and Adaptive Resonance Theory (ART) are considered for classifiers. The main advantage of ART is that the number of clusters does not need to be specified a priori as in K-means. Rather a vigilance parameter is set to control the formation of a new cluster and is dependent on the data being clustered. More detail can be found
                        <a href="Publications/Neural_Network_Classification.pdf" font-weight="bold"><b>here</b></a>.
                    </p>
                </div>
            </div>
        </article>

        <article id="benchmark-blast">
            <h3>Benchmarking Accuracy of BLAST Taxonomic Classification of Metagenomics Reads</h3>
            <div class="project-content">
                <figure class="project-figure">
	                <a href="Posters/BLAST_RECOMB.pdf">
		                <img src="images/benchmark-blast.png" alt="BLAST Benchmark Poster">
                    </a>
                </figure>
                <div class="project-description">
                    <p>
                        Classification of novel DNA sequences to bacterial taxonomy is an integral task for researchers in numerous areas of biology. <a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" target="_blank">BLAST</a>, a supervised learning method, has widespread use in searching for closest matches based on sequence similarity to known organisms. More than 95% of bacteria are not in the databases, however. While effective on known organisms, it is unknown how accurate BLAST classification is on novel sequences. This study investigates BLAST accuracy on datasets with under-represented genomes in the database and errors in the query sequences. More detail can be found <a href="Publications/Benchmarking_BLAST.pdf" font-weight="bold" target="_blank"><b>here</b></a> and <a href="Publications/Sequence_Error_BLAST.pdf" font-weight="bold" target="_blank"><b>here</b></a>. The study has also been expanded to investigate other methods such as the Naive Bayes Classifier, <a href="Publications/Comparison_Statistical_Metagenomics.pdf" font-weight="bold" target="_blank"><b>here</b></a>.
                    </p>
                </div>
            </div>
        </article>

        <article id="novodisplay">
            <h3>Wireless Mesh-Networked E-Signage System</h3>
            <div class="project-content">
                <figure class="project-figure">
                    <a href="Publications/NOVOdisplay_ESign_Platform_Whitepaper.pdf">
                        <img src="images/novo-system.png" alt="Wireless System Diagram">
                    </a>
                </figure>
                <div class="project-description">
                    <p>
                        While working in the R&D department at Metrologic (Honeywell) I worked up to a team lead position, developing an electronic paper retail signage system supported by a Zigbee mesh-network communication system. I authored a brief white paper describing the product, which may be found <a href="Publications/NOVOdisplay_ESign_Platform_Whitepaper.pdf" font-weight="bold" target="_blank"><b>here</b></a>. Working in our 10 person globally-distributed team we developed several working prototypes for potential customers and generated numerous patents based on our work. Relevant patents may be found <a href="http://appft.uspto.gov/netacgi/nph-Parser?Sect1=PTO2&Sect2=HITOFF&u=%2Fnetahtml%2FPTO%2Fsearch-adv.html&r=0&p=1&f=S&l=50&Query=in%2Fessinger+and+an%2Fmetrologic+and+abst%2Fwireless&d=PG01" font-weight="bold" target="_blank"><b>here</b></a> and <a href="http://patft.uspto.gov/netacgi/nph-Parser?Sect1=PTO2&Sect2=HITOFF&u=%2Fnetahtml%2FPTO%2Fsearch-adv.htm&r=0&p=1&f=S&l=50&Query=in%2Fessinger+and+an%2Fmetrologic+and+abst%2Fwireless&d=PTXT" font-weight="bold" target="_blank"><b>here</b></a>. My direct responsibilities included the systems design and drafting of the figures describing the system, developing IP, the RF hardware design, coordination of all technical development and supporting marketing and sales with customer demonstrations.
                    </p>
                </div>
            </div>
        </article>
    </section>

    <footer>
        <p>© Copyright 2014-2024 Steve Essinger</p>
        <p>This site was built in HTML by hand. Inspired by <a href="https://brutalist-web.design/">Brutalist Web Design</a></p>
    </footer>
</div>
</body>
</html>
